plugins/inputs/socket_listener/socket_listener_test.go

```diff
@@ -40,7 +39,6 @@ func TestSocketListener_udp(t *testing.T) {
 }
 
 func TestSocketListener_unix(t *testing.T) {
-	defer os.Remove("/tmp/telegraf_test.sock")
```

Comment:
Dunno how I forgot about the issue when I explicitly put the removal of the file in the test code...